views/default/page/elements/foot.php

```diff
@@ -1,6 +1,9 @@
 <?php
 
-$elgg_init = elgg_view('initialize_elgg.js');
+// load special js to prevent certain clicks
+$elgg_init = elgg_view('elgg/prevent_clicks.js');
```

Comment:
or should i do this with elgg_extend_view('initialize_elgg', 'prevent_clicks', 1); ??